VAULT-2285 adding capability to accept comma separated entries for au… (#12126)
hashicorp/vault: website/content/docs/commands/auth/tune.mdx

The following flags are available in addition to the [standard set of
flags](/docs/commands) included on all commands.

> - `-allowed-response-headers` `(string: "")` - response header values that plugins

Reviewer: ```suggestion
- `-allowed-response-headers` `(string: "")` - response header values that the auth method
```

  configured default lease TTL, or a previously configured value for the auth
  method.

- `-description` `(string: "")` - Specifies the description of the mount.
  This overrides the current stored value, if any.

> - `-max-lease-ttl` `(duration: "")` - The maximum lease TTL for this auth

Reviewer: Add listing-visibility?

Author: Done

  method. If unspecified, this defaults to the Vault server's globally
  configured maximum lease TTL, or a previously configured value for the auth
  method.

- `-passthrough-request-headers` `(string: "")` - request header values that will

>   be sent to the plugin.

Reviewer: ```suggestion
  be sent to the auth method.
```

- `-token-type` `(string: "")` - Specifies the type of tokens that should be

>   returned by the mount.

Reviewer: ```suggestion
  returned by the auth method.
```
